scene/2d/path_2d.cpp

```diff
@@ -329,7 +329,7 @@ void PathFollow2D::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "h_offset"), "set_h_offset", "get_h_offset");
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "v_offset"), "set_v_offset", "get_v_offset");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "rotates"), "set_rotates", "is_rotating");
-	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "cubic_interp"), "set_cubic_interpolation", "get_cubic_interpolation");
+	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "cubic_interpolation"), "set_use_cubic_interpolation", "is_using_cubic_interpolation");
```

Comment:
Maybe the property should be `use_cubic_interpolation`?

Reply:
I think it's redundant. I've seen members without the use_ prefix elsewhere. e.g. `Particles::local_coords`, `RigidBody3D::custom_integrator` and etc. but it may be minority.

Reply:
The rule normally is that the property name and the setter/getter should match. If the property is "name", the setter is "set_name", and the getter is "get_name".

For booleans it's a bit trickier, but it's still better to follow this pattern if possible (sometimes like here with "is" + "-ing" instead of "get").

Ideally this would be `interpolation_mode` with an enum, but well, that's out of scope.

Reply:
Changed (looks a little longer...)

![image](https://user-images.githubusercontent.com/61938263/217346294-82e570a1-4221-4370-8580-746a34dabe7d.png)